chime.py

```diff
@@ -17,7 +19,48 @@
     IPYTHON_INSTALLED = False
 
 
-THEME = 'chime'  # default theme
+def _get_config_path(system: str) -> pathlib.Path:
+    """Return the path of the config file.
+
+    Parameters:
+        system: The OS being used.
+    """
+    home_dir = pathlib.Path.home()
+    if system == "Windows":
+        config_path = (
+            pathlib.Path(
+                         os.getenv('APPDATA',
+                                   home_dir / pathlib.Path("AppData", "Roaming")))
```

Comment:
Can you use single quotes instead of double quotes? I've forgotten what's PEP8 but I guess we should align.

Reply:
Good catch! Tried to match your code style as much as possible but some things slipped. Feel free to let me know if other things look off!